src/TAlias.cpp

```diff
@@ -89,13 +89,7 @@ bool TAlias::match(const QString& haystack)
         return false; //regex compile error
     }
 
-#if defined(Q_OS_WINDOWS)
-    // strndup(3) - a safe strdup(3) does not seem to be available on mingw32 with GCC-4.9.2
-    char* haystackC = static_cast<char*>(malloc(strlen(haystack.toUtf8().constData()) + 1));
-    strcpy(haystackC, haystack.toUtf8().constData());
-#else
     char* haystackC = strndup(haystack.toUtf8().constData(), strlen(haystack.toUtf8().constData()));
```

Comment:
This is still needed - as the build failure for https://github.com/Mudlet/Mudlet/actions/runs/13240914186/job/36955894150?pr=7701 shows!
```suggestion
#if defined(Q_OS_WINDOWS)
    // strndup(3) - a safe strdup(3) does not seem to be available in the
    // original Mingw or the replacement Mingw-w64 environment we use:
    char* haystackC = static_cast<char*>(malloc(strlen(haystack.toUtf8().constData()) + 1));
    strcpy(haystackC, haystack.toUtf8().constData());
#else
    char* haystackC = strndup(haystack.toUtf8().constData(), strlen(haystack.toUtf8().constData()));
#endif
```